testing/esp/esdlcmd/baselines/wstest-cpp-installdir/WsTestServiceBase.hpp

```diff
@@ -0,0 +1,167 @@
+#ifndef WSTESTSERVICEBASE_HPP__
+#define WSTESTSERVICEBASE_HPP__
+
+#include "jlib.hpp"
+#include "jptree.hpp"
+#include "jarray.hpp"
+#include "primitivetypes.hpp"
+
+using namespace std;
+using namespace cppplugin;
+
+class EsdlContext : public CInterface, implements IInterface
```

Comment:
Implementing IInterface is only necessary if you intend to treat the instance as an IInterface (which is not necessary for use with Owned). Is there a reason for inheriting both CInterface and IInterface in so many places?

Reply:
There may be some opportunities to improve the auto-generated code, but I'm not going to address it as part of this ticket.